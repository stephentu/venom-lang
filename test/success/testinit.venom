class A 
  attr x::int = 10
  attr y::string = "foobar"
  attr z::int = 1
  def self(z::int) =
    self.z = z;
  end
end
print(A(3).x);
print(A(3).y);
print(A(3).z);
