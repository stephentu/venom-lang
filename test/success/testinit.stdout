10
foobar
3
